symphony/lib/toolkit/class.field.php

```diff
@@ -1538,10 +1538,25 @@ public function buildDSRetrievalSQL($data, &$joins, &$where, $andOperation = fal
     }
 
     /**
+     * Determine if the requested $order is random.
+     *
+     * @since Symphony 2.7.0
+     * @param string $order
+     *  the sorting direction.
+     * @return boolean
+     *  true if the $order is either 'rand' or 'random'
+     */
+    public static function isRandomOrder($order)
```

Comment:
static: because it does not uses `$this` and it does not have a state. Why would it need to be an instance method ? I've yet to find a good reasoning on this, any help is appreciated!

public: to be able to unit test it, one day...